### PR TITLE
Only use POST for method to save to RC

### PR DESCRIPTION
**Pull request by stulees**
- Electron has an issue when it sends redirect request via custom scheme protocol handling and the method is PUT. Working around this by always using POST.

### CODE REVIEW

**Review by rodrigopavezi — APPROVED** [review]
@stulees looks good 👍  cheers

File: rise-data.html [hunk 1]
```diff
@@ -120,7 +120,7 @@
       _getCacheUrl: function( key, method ) {
         var url = this._baseCacheUrl + "/" + this.endpoint;
 
-        if ( method === "PUT" || method === "DELETE" || method === "GET" ) {
+        if ( method === "DELETE" || method === "GET" ) {
```
Comment (tejohnso) [inline rise-data.html:123]: What's happening here?

Comment (stulees) [reply rise-data.html:123]: @tejohnso Removing "PUT" from acceptable methods for appending the `"/" + key` to the url below.

Comment (tejohnso) [reply rise-data.html:123]: So if an invalid method is requested it's going to continue as though the request was made with no key?  Then what?  Why aren't we returning a 405?

Comment (stulees) [reply rise-data.html:123]: @tejohnso This function `_getCacheUrl` just returns a constructed Rise Cache url for making the request. This component now only applies 3 different CRUD methods to RC requests - POST, GET, and DELETE, and the only method that doesn't require the key is POST as it provides a body to the request instead. RC `spreadsheets` endpoint is designed to handle those 3 request methods accordingly, so there will never be a scenario where RC would return a 405 specifically.

Comment (tejohnso) [reply rise-data.html:123]: Okay

**Review by tejohnso — APPROVED** [review]

**Pull request merged**